spec/bundler/stub_specification_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+require "spec_helper"
+
+if Bundler.rubygems.provides?(">= 2.1")
+  RSpec.describe Bundler::StubSpecification do
+    let(:with_gem_stub_spec) do
+      stub = Gem::Specification.stubs.first
```

Comment:
watch out, this is returning `nil` in certain cases on CI... I'd create a new spec, serialize it to disk, and then create a stub from that known spec